packages/opentelemetry-plugin-http/src/utils.ts

```diff
@@ -122,7 +121,7 @@ export const hasExpectHeader = (options: RequestOptions): boolean => {
   }
 
   const keys = Object.keys(options.headers);
-  return !!keys.find(key => key.toLowerCase() === 'expect');
+  return !!keys.find((key) => key.toLowerCase() === 'expect');
```

Comment:
arrow functions with only one argument should not require ()

Reply:
we have two options:

```
"as-needed" enforces no braces where they can be omitted.
Object properties for variants of the "as-needed" option:

"requireForBlockBody": true modifies the as-needed rule in order to require parens if the function body is in an instructions block (surrounded by braces).
```

ref: https://eslint.org/docs/rules/arrow-parens

I would prefer requireForBlockBody but as-needed is ok.

Reply:
I prefer as-needed. I don't understand the advantage of requireForBlockBody